dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/data/privacy/ConsentProvider.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.core.internal.data.privacy
+
+import com.datadog.tools.annotation.NoOpImplementation
+
+@NoOpImplementation
+internal interface ConsentProvider {
+
+    fun getLastConsent(): Consent
```

Comment:
Nitpicking here, you can just call this method `getConsent`. `getLastConsent` would make me think that there's another consent than the last.